src/components/Form/Form.stories.tsx

```diff
@@ -2,7 +2,7 @@ import type { Meta, StoryObj } from '@storybook/react';
 import { RJSForm } from '.';
 
 const meta = {
-	title: 'Mui Components/Form',
+	title: 'Other/RJSX Form',
```

Comment:
```suggestion
	title: 'Other/RJS Form',
```

Reply:
It's react-json-schema-form, the JS are not intended to stand for javascript or JSX 😅

Reply:
haha nice catch @myarmolinsky :)